x-pack/plugins/uptime/public/components/overview/alerts/monitor_expressions/filters_expression_select.test.tsx

```diff
@@ -52,125 +70,63 @@ describe('filters expression select component', () => {
-              onClick={[Function]}
-            />
-          </EuiFlexItem>
-          <EuiSpacer
-            size="xs"
-          />
-        </EuiFlexGroup>
-        <EuiSpacer
-          size="xs"
-        />
-      </Fragment>
-    `);
+    expectedLabels.forEach((label) => expect(getByLabelText(label)));
+    absentLabels.forEach((label) => expect(queryByLabelText(label)).toBeNull());
   });
 
-  it('contains provided selected filter values', () => {
-    const component = shallowWithIntl(
-      <FiltersExpressionsSelect
-        alertParams={{}}
-        newFilters={['tags']}
-        onRemoveFilter={jest.fn()}
-        filters={{
-          tags: ['foo', 'bar'],
-          ports: [],
-          schemes: [],
-          locations: [],
-        }}
-        setAlertParams={jest.fn()}
-        setUpdatedFieldValues={jest.fn()}
-        shouldUpdateUrl={false}
-      />
-    );
-    expect(component).toMatchInlineSnapshot(`
-      <Fragment>
-        <EuiFlexGroup
-          key="filter_tags"
-        >
-          <EuiFlexItem>
-            <FilterPopover
-              btnContent={
-                <EuiExpression
-                  aria-label="ariaLabel"
-                  color="secondary"
-                  data-test-subj="uptimeCreateStatusAlert.filter_tags"
-                  description="Using"
-                  onClick={[Function]}
-                  value="any tag"
-                />
-              }
-              disabled={false}
-              fieldName="tags"
-              forceOpen={false}
-              id="filter_tags"
-              items={
-                Array [
-                  "foo",
-                  "bar",
-                ]
-              }
-              loading={false}
-              onFilterFieldChange={[Function]}
-              selectedItems={Array []}
-              setForceOpen={[Function]}
-              title="Tags"
-            />
-          </EuiFlexItem>
-          <EuiFlexItem
-            grow={false}
-          >
-            <EuiButtonIcon
-              aria-label="Remove filter"
-              color="danger"
-              iconType="trash"
-              onClick={[Function]}
-            />
-          </EuiFlexItem>
-          <EuiSpacer
-            size="xs"
-          />
-        </EuiFlexGroup>
-        <EuiSpacer
-          size="xs"
+  it.each([
+    [
+      {
+        tags: ['foo', 'bar'],
+        ports: [],
+        schemes: [],
+        locations: [],
+      },
+      filterAriaLabels.TAG,
+    ],
+    [
+      {
+        tags: [],
+        ports: [5601, 9200],
+        schemes: [],
+        locations: [],
+      },
+      filterAriaLabels.TAG,
+    ],
+    [
+      {
+        tags: [],
+        ports: [],
+        schemes: ['http', 'tcp'],
+        locations: [],
+      },
+      filterAriaLabels.TAG,
+    ],
+    [
+      {
+        tags: [],
+        ports: [],
+        schemes: [],
+        locations: ['nyc', 'fairbanks'],
+      },
+      filterAriaLabels.TAG,
```

Comment:
will it be `filterAriaLabels.TAG` in all cases?

Reply:
Yes, I named it here: https://github.com/elastic/kibana/pull/88185/files#diff-80de884962d2ef2481e6a15d7a672f2f2aac0daad2c10279492c8be6a2157079R94

I also changed the translation from `'tags'` to `'tag'` because I felt it should match the other fields, which are like `scheme`, `port`, etc.; this didn't seem to matter as much until I started relying on the value to build a11y labels.

Reply:
Sorry, I misunderstood your ask. I will address this.